modules/dcache-nfs/src/main/java/org/dcache/chimera/nfsv41/door/NFSv41Door.java

```diff
@@ -848,11 +843,23 @@ public String call() throws IOException {
             description = "Show active transfers excluding proxy-io.")
     public class ShowTransfersCmd implements Callable<String> {
 
+        @Option(name = "pool")
```

Comment:
The `@Option` annotation has a `usage`  field.  Although this is optional, the default value (an empty string) means that these options will not be include in the online documentation, which would be a shame.

Perhaps you could add a short `usage` field for each option?  It doesn't have to be long, one sentence is enough, but it would be nice if the message mentioned that glob pattern matching is supported.

Here's an example on how to see the online help:
```
paul@celebrimbor:~/git/dCache (master)$ ssh -p 22224 admin@localhost
dCache (3.2.0-SNAPSHOT)
Type "\?" for help.

[celebrimbor] (local) admin > \c NFS-celebrimbor 
[celebrimbor] (NFS-celebrimbor@dCacheDomain) admin > \h show transfers
NAME
       show transfers -- show active transfers

SYNOPSIS
       show transfers 

DESCRIPTION
       Show active transfers excluding proxy-io.


[celebrimbor] (NFS-celebrimbor@dCacheDomain) admin > 
```